.github/workflows/build.yml

```diff
@@ -25,11 +25,10 @@ jobs:
           key: ${{ runner.os }}-ztl-validate-cached-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('**/composer.lock') }}-v1
           restore-keys: |
             ${{ runner.os }}-ztl-validate-cached-
-
       - name: Setup composer and install packages
         uses: php-actions/composer@v6
         with:
-          php_version: 8.0.22
+          php_version: "8.1"
```

Comment:
I don't think this needs quotes - I'd like to leave them out if they aren't necessary to keep the workflow files consistent

Reply:
https://github.com/marketplace/actions/composer-php-actions
"Make sure to put the PHP version number in quotes, otherwise YAML will interpret e.g. 8.0 as 8 which means latest 8.x, not 8.0."

Reply:
Interesting - I didn't know that. Good to know!